src/controllers/activegate/reconciler/statefulset/statefulset.go

```diff
@@ -45,6 +48,7 @@ type statefulSetProperties struct {
 	initContainersTemplates []corev1.Container
 	containerVolumeMounts   []corev1.VolumeMount
 	volumes                 []corev1.Volume
+	log                     logr.Logger
```

Comment:
this should still be removed in this pr, even if it'll also be removed later

Reply:
OK, no problem.